libraries/creative-renderer-display/renderer.js

```diff
@@ -1,2 +1,2 @@
 // this file is autogenerated, see creative/README.md
-export const RENDERER = "!function(){\"use strict\";window.render=function({ad:d,adUrl:i,width:n,height:e},{mkFrame:o},r){if(!d&&!i)throw{reason:\"noAd\",message:\"Missing ad markup or URL\"};{const t=r.document,s={width:n,height:e};i&&!d?s.src=i:s.srcdoc=d,t.body.appendChild(o(t,s))}}}();"
\ No newline at end of file
+export const RENDERER = "(()=>{\"use strict\";window.render=function({ad:d,adUrl:e,width:i,height:r},{mkFrame:n},o){if(!d&&!e)throw{reason:\"noAd\",message:\"Missing ad markup or URL\"};{const s=o.document,t={width:i,height:r};e&&!d?t.src=e:t.srcdoc=d,s.body.appendChild(n(s,t))}}})();"
```

Comment:
lgtm, just undo your changes to this file and the below, these should be autogenerated and not modified by your pr

Reply:
ooh sorry . I have reverted that changes.